### PR TITLE
norm-hvg first draft

### PR DESCRIPTION
**Pull request by MarcElosua**
First draft of the normalization and HVG notebook

### CODE REVIEW

**Comment by kylekimler** [comment]
Checked offline! It knits correctly just 2 spelling mistakes 
line 58 its -> it's
line 39 Exampled -> examples

File: beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+2) Feature selection is necessary to reduce the dimensionality of the dataset while preserving as much biological information as possible. Highly variable genes are selected following the assumption that genes that vary throughout the data will represent different cell types. Genes that have a homogeneous expression are less interesting.
+
+
+### Glossay
+
+- **Unique molecular identifier (UMI)**: unique molecular barcode added to each transcript before PCR amplification to correct for biases during that process.
+
+- **Library size**: total number of UMIs detected for each cell.
+
+- **Library complexity**: number of genes that have expression > 0 in each cell. The more genes detected in a cell the more complex a cell is. Exampled of low complexity cells are platelets and red blood cells which can be confounded with low quality cells if not assessed properly.
```
Comment (kylekimler) [inline beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd]:
```suggestion
- **Library complexity**: number of genes that have expression > 0 in each cell. The more genes detected in a cell the more complex a cell is. Examples of low complexity cells are platelets and red blood cells which can be confounded with low quality cells if not assessed properly.
```

Comment (MarcElosua) [inline beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd]:
```suggestion
- **Library complexity**: number of genes that have expression > 0 in each cell. The more genes detected in a cell the more complex a cell is. Examples of low-complexity cells are platelets and red blood cells which can be confounded with low-quality cells if not assessed properly.
```

File: beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    
+    2. Log transformation mitigates the mean-variance relationship
+    
+    3. Reduces the skewness of the data fulfilling the assumptions of downstream tools
+
+-   Feature selection is the first step to reduce the dimensionality of our data.
+
+-   We select highly variable genes (HVGs) following the assumption that genes that have variable expression are representative of cellular heterogeneity within our data.
+
+-   Feature selection using HVGs is a KEY step in the analysis workflow. It determines the biological information universe we will be using for downstream analysis and is dependent on the data used. Every time we subset the data we have to recompute HVG to update our biological universe so its representative of the new cellular context!
```
Comment (kylekimler) [inline beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd]:
```suggestion
-   Feature selection using HVGs is a KEY step in the analysis workflow. It determines the biological information universe we will be using for downstream analysis and is dependent on the data used. Every time we subset the data we have to recompute HVG to update our biological universe so it's representative of the new cellular context!
```

Comment (MarcElosua) [inline beginners-guide-to-analyzing-scRNAseq/day-1/3-norm-hvg.qmd]:
```suggestion
-   Feature selection using HVGs is a KEY step in the analysis workflow. It determines the biological information universe we will be using for downstream analysis and is dependent on the data used. Every time we subset the data we have to recompute HVG to update our biological universe so it's representative of the new cellular context!
```

**Pull request merged**